Vizzuality/gfw
# Feature/temp aoi dashboard

![Screenshot 2019-08-27 at 18 20 32](https://user-images.githubusercontent.com/20288774/63789325-67797080-c8f7-11e9-801f-cf927456fa6e.png)

Adds the pending message for dashboards for both public and private view. When a user clicks "got it!" the message disappears. 

Testing:
- Go to both a public and private area and check the messages are differnet
- Click to hide message
